src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -149,11 +151,14 @@ public void execute(JobExecutionContext jobExecutionContext) {
     if (isActionableJob) {
       if (userAccountDetails != null) {
         jobDetails.putIfAbsent("isRegistered", userAccountDetails.isRegistered());
-        jobDetails.putIfAbsent("title", userAccountDetails.title());
-        jobDetails.putIfAbsent("familyName", userAccountDetails.familyName());
-        jobDetails.putIfAbsent("givenName", userAccountDetails.givenName());
+        jobDetails.putIfAbsent("title", capitalizeFully(userAccountDetails.title()));
```

Comment:
I think we decided to leave these as-is and not try to fix the capitalisation. The reason is that (1) it's how the trainee themselves provided their own name and (2) its really tricky 😂  (for example, family name 'du Pont' or given name 'MaryAnne', or title 'Lt Col')